ansible_ai_connect_chatbot/src/App.test.tsx

```diff
@@ -175,6 +183,38 @@ describe("App tests", () => {
 
     const sureButton = screen.getByText("Sure!");
     await act(async () => fireEvent.click(sureButton));
+
+    expect(ghIssueLinkSpy.mock.calls.length).toEqual(1);
+    expect(ghIssueLinkSpy.mock.calls[0][0]?.toString().length).toEqual(477);
```

Comment:
It'd be nice to explain this m-a-g-i-c number either in a comment.

Alternatively if it's `referencedDocumentExample.length` please use it!

(or the length of ascii7 encoded `referencedDocumentExample` etc)

Reply:
good catch, it was tricky... I have improved the test cases, please take a look at let me know if they're easier to follow. Thx